packages/request-client.js/src/api/request.ts

```diff
@@ -695,10 +695,9 @@ export default class Request {
    */
   public async refreshBalance(): Promise<Types.Payment.IBalanceWithEvents<any> | null> {
     // TODO: PROT-1131 - add a pending balance
-    this.balance =
-      this.paymentNetwork && this.requestData
-        ? await this.paymentNetwork.getBalance(this.requestData)
-        : this.balance;
+    if (this.paymentNetwork && this.requestData) {
```

Comment:
I don't think the `feeBalance` was included in the `balance` object, no... 

Actually, changes on this files are because of an attempt to clean the adding of `feeBalance`, but it's not that easy

Reply:
Yeah, right, not in `balance`, but in the extension values. That change is for another day anyway